### ISSUE TITLE
Code Table Request - Change IF: Idaho Frozen Tissue Collection to IF

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6373]
## Initial Request

_**Goal**: the identifier "IF: Idaho Frozen Tissue Collection" was created to be a functional equivalent to the identifiers "AF" and "NK", as each of these IDs refer to frozen tissues collected by the Cook lab at various institutions. I am requesting that the IF value be converted to the same format as the other two equivalent identifiers so that they can be treated and searched on in exactly same way. Currently, only AF and NK are available in the Search Results options - once this process is complete I will file a separate issue to have IF added to the Search Results page. These identifiers are critical to linking samples, hosts, and parasites across multiple institutions, and they need to be readily accessible and interchangeable as identifiers for that purpose.

_**Proposed Value**:  
"IF"

_**Proposed Definition**: The IF number was assigned to mammals, parasites, and tissues by the Joseph A. Cook lab during his tenure at University of Idaho to serve as an institutional field number and linking identifier for all primary and derivative specimens, tissues, samples, and data. This identifier is functionally identical to the identifiers "NK" and "AF" used for the same purpose at the Museum of Southwestern Biology and the University of Alaska Fairbanks Museum of the North, assigned by the same researcher for the same purpose. 

_**Context**: Describe why this new value is necessary and existing values are not._
This value will replace the current value "IF:Idaho Frozen Tissue Collection", which is not only unnecessarily long and not in the same format as the other related identifiers, but also refers to an Idaho frozen tissue collection which does not exist; all these materials are actually part of Cook lab research materials that are now at the University of New Mexico. The identifier links specimens and tissues deposited at UAM and MSB. There are no specimens and tissues from these collections deposited in Idaho, frozen or otherwise. In order to search and manage related materials linked by the identifiers NK, AF, and IF at UAM and MSB, we need all three identifiers in the same format and treated as functional equivalents for search and download.

_**Table**:  https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type

_**Priority**: High. Needed for grant funded work, upcoming presentation, and identifier code table cleanup.

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
The definition needs work. Perhaps:

>The IF number was created and assigned by the Joseph A. Cook lab during his tenure at University of Idaho.

Adding dates might be useful.

??

**Comment by campmlc** [comment]
2000-2003

**Comment by campmlc** [comment]
Thanks!!
